### PR TITLE
Summaries

### PR DESCRIPTION
**Pull request by jwscook**
Addition of summaries provided in dropdowns from the links to the pdfs.

### CODE REVIEW

File: docs/source/conf.py [hunk 1]
```diff
@@ -148,7 +154,7 @@ def clean_title(title):
         title = re.sub(r"\b%s\b" % i, "", title, re.I)
     title = title.title()  # make first letter of all words uppercase
     title = re.sub(r"\W+", "", title)  # strip out any non alphanumeric values
-    title = title.replace(" ", "")  #  remove all spaces
+#    title = title.replace(" ", "")  #  remove all spaces
```
Comment (oparry-ukaea) [inline docs/source/conf.py]: Remove commented line?

File: docs/source/conf.py [hunk 2]
```diff
@@ -198,21 +217,26 @@ def copy_and_rename_file(pdfpath, pdftxt):
...
         if key in str(dirstub):
-            new_file_name = dirstub.replace(key, numbered_to_descriptive_dirs[key])
+            new_file_name = dirstub.replace(key, "")[1:]
             break
     if new_file_name is None:
         return None
     # add descriptive title-like suffix
     readable_suffix = "_" + clean_title(descriptive_title_from_pdf(pdfpath, pdftxt))
     new_file_name = new_file_name[:-4] + readable_suffix + ".pdf"
+    #new_file_name = readable_suffix + ".pdf"
```
Comment (oparry-ukaea) [inline docs/source/conf.py]: Remove commented line?

**Review by oparry-ukaea — APPROVED** [review]
All looks fine to me.
Not a big issue, but I wonder if we could get it to run through all the jsons and unify keywords with small variations?
e.g.
"ExCAL-IBUR NEPTUNE"
"Excalibur-Neptune"
"ExCALIBUR-Neptune"
...

File: docs/source/conf.py (hunk 2 above)
Comment (jwscook) [reply docs/source/conf.py]: pushed a deletion

File: docs/source/conf.py (hunk 1 above)
Comment (jwscook) [reply docs/source/conf.py]: pushed a deletion

**Comment by jwscook** [comment]
> All looks fine to me. Not a big issue, but I wonder if we could get it to run through all the jsons and unify keywords with small variations? e.g. "ExCAL-IBUR NEPTUNE" "Excalibur-Neptune" "ExCALIBUR-Neptune" ...

Yup - I've pushed changes

**Review by oparry-ukaea — APPROVED** [review]
:+1: Looks good to me

**Review by oparry-ukaea — APPROVED** [review]

**Pull request merged**